src/masks/InteractionMasks.tsx

```diff
@@ -241,7 +241,7 @@ export default function InteractionMasks<R, SR>({
       cellKey,
       fromRow,
       toRow,
-      updated: { [cellKey]: value } as never,
+      updated: { [cellKey]: value },
```

Comment:
Since the update is not tied to the row key, I'm wondering if we should make this loosen?

Reply:
Not sure I follow? Can you give an example?

Reply:
Ignore this, I thought it was something else.